dialogue/TownNPCs/ShovelGuy/ShovelGuy1.json

```diff
@@ -14,7 +14,7 @@
       "And it involves my back."
     ],
     "pl": [
-      "<c=mocha>Mocha</c> poprosiła mnie, żebym wykopał trochę miejsca na magazynę, który ona planuje zbudować.",
+      "<c=mocha>Mocha</c> poprosiła mnie, żebym wykopał trochę miejsca na magazyn, który ona planuje zbudować.",
```

Comment:
You can delete "ona" after "który", because it's already clear who this is about